src/Security/MemberAuthenticator/MemberLoginForm.php

```diff
@@ -132,9 +131,6 @@ protected function getFormFields()
         $label = Member::singleton()->fieldLabel(Member::config()->get('unique_identifier_field'));
         $fields = FieldList::create(
             HiddenField::create("AuthenticationMethod", null, $this->getAuthenticatorClass(), $this),
-            // Regardless of what the unique identifier field is (usually 'Email'), it will be held in the
-            // 'Email' value, below:
```

Comment:
These were not part of the todo comment - they're adding useful context that this isn't _necessarily_ actually an email address. Please bring this comment back.

Reply:
Updated